aip/general/0145.md

```diff
@@ -48,6 +48,25 @@ Exclusive end values are preferable for the following reasons:
 - It is easier to reason about abutting ranges: `[0, x), [x, y), [y, z)`, where
   values are chainable from one range to the next.
 
+### Use of `google.type.Interval`
+
+The [`google.type.Interval`][interval] type (in the common protos described in
+[AIP-213][]) represents a range between two timestamps, with an inclusive start
+value and exclusive end value as described above.
+
+Ranges between two timestamps which conform to the expectations of the
+`Interval` message **should** usually use this rather than having separate
+start and end fields. This allows client code to be written against the
```

Comment:
I think this can be read easier if written as follows:

> Client code can check whether a given timestamp occurs within the `Interval`. Further, declared intervals can be used across the same API or even across multiple APIs.

Reply:
No, that's changing the meaning. The point isn't that you're sharing the "declared intervals" - it's that you can write common code which works across any interval.

Client code can check whether a given timestamp occurs within the range if you have separate fields, too - it's just that that code can't be as common (or would have to take two parameters).